common/src/main/cpp/extensions/openxr_fb_body_tracking_extension_wrapper.cpp

```diff
@@ -0,0 +1,359 @@
+/**************************************************************************/
+/*  openxr_fb_body_tracking_extension_wrapper.cpp                         */
+/**************************************************************************/
+/*                       This file is part of:                            */
+/*                              GODOT XR                                  */
+/*                      https://godotengine.org                           */
+/**************************************************************************/
+/* Copyright (c) 2022-present Godot XR contributors (see CONTRIBUTORS.md) */
+/*                                                                        */
+/* Original contributed implementation:                                   */
+/*   Copyright (c) 2024 Malcolm Nixon                                     */
```

Comment:
@Malcolmnixon We follow this approach for 'code donation' by third party, but we'd want to disincentivize this for contributions by team members.

cc @BastiaanOlij

Reply:
Removed these from source and header files.